include/buffer_control_block.h

```diff
@@ -10,8 +10,11 @@
 #include <string>
 #include <mutex>
 #include <condition_variable>
+#include "update.h"
 
 typedef uint32_t buffer_id_t;
+typedef uint32_t File_Pointer;
+typedef std::pair<Node, buffer_id_t> work_t;
```

Comment:
what does this type signify?

Reply:
File_Pointer is an index into a file.
work_t is the type we will put in the work queue and identifies a node/location to pull data from.